packages/bierzo-wallet/src/routes/registerName/components/index.tsx

```diff
@@ -156,7 +156,7 @@ const Layout = ({
         <Button
           fullWidth
           type="submit"
-          disabled={invalid || pristine || submitting || validating}
```

Comment:
Why?

Reply:
Unfortunately if I remove address line then internal state of the form component  not changing. So it is not possible to update the name because button is disabled.